odoo/modules/registry.py

```diff
@@ -404,8 +405,7 @@ def check_signaling(self):
             # Check if the model caches must be invalidated.
             elif self.cache_sequence != c:
                 _logger.info("Invalidating all model caches after database signaling.")
-                self.clear_caches()
-                self.cache_invalidated = False
+                self.clear_caches(invalidate=False)
```

Comment:
Do you think we could make the same fix without changing the signature of the clear_caches methods? Like directly call `self.cache.clear()` instead of `self.clear_caches()` here? Or did you encounter issue trying that?

I know this sounds less extensible / modular, but we're talking of version 11 here, and I'd rather not change API signatures if we can avoid it. We can always do that in master if it's worth it.

Reply:
Alright, I hadn't dug deeper to check that the `pool` for each value from `self.models` corresponds to the same `Registry` but it seems to be the case indeed (at least in standard).

The difference here will be that if someone overrode a `model.clear_caches` we won't go through it anymore, I'm not sure which is cleaner.

Reply:
> The difference here will be that if someone overrode a `model.clear_caches` we won't go through it anymore, I'm not sure which is cleaner.

Yup, that's what I meant with "less extensible/modular", but I don't think it's worth changing API sigs in 11 for that. We can do that later in master (i.e. tell fw-bot to stop to 13.1 and do a separate PR for master)